portfolio/src/main/java/com/google/sps/servlets/FormServlet.java

```diff
@@ -0,0 +1,62 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.Arrays;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.Date;
+
+import java.util.ArrayList;
+
+/** Servlet that processes text. */
+@WebServlet("/form")
+public final class FormServlet extends HttpServlet {
+    String textForm;
+    String rateValueForm;
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = getParameter(request, "text-input", "");
+    String rates = request.getParameter("color-choise");
+    String rate_value = request.getParameter("color");
+    if ("red".equals(rate_value)) {
```

Comment:
This is what we call a "yoda" condition: if you were speaking in English you would say "if the rate_value is red", but Yoda would say "if red is the rate_value". For that reason, I'd recommend writing `if (rate_value.equals("red"))` instead :)